### PR TITLE
PS-547-4 Add J1939 PGN to device tags

### PR DESCRIPTION
**Pull request by FelixTing**

### CODE REVIEW

File: central/dbc/transform.go [hunk 1]
```diff
@@ -125,7 +126,11 @@ func ConvertDBCtoDevice(data []byte, networkName, serviceName string) (deviceDTO
 					Sender:   m.SenderNode,
 				},
 			},
+			Tags: map[string]interface{}{
```
Comment (cloudxxx8) [inline central/dbc/transform.go:129]: use `any` instesd of `interface{}`

File: central/dbc/transform_test.go [hunk 2]
```diff
@@ -52,6 +52,9 @@ func TestConvertDBCtoDevice(t *testing.T) {
 				Sender:   "Vector__XXX",
 			},
 		},
+		Tags: map[string]interface{}{
```
Comment (cloudxxx8) [inline central/dbc/transform_test.go:55]: use `any` instead of `interface{}`

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: central/dbc/transform.go (hunk 1 above)
Comment (FelixTing) [reply central/dbc/transform.go]: modified

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**